easy/3.js

```diff
@@ -1,11 +1,30 @@
 function minMax(arr) {
-	
+    let howMany = arr.length;

```

Comment:
Read about function eg. Math.min()
World would be worse without built-in function in JS global objects :D